currency/l1/public.yml

```diff
@@ -159,3 +160,133 @@ paths:
             text/html:
               schema:
                 type: string
+
+  /data/estimate-fee:
+    post:
+      tags:
+        - Data
+      description: Estimate fee for given data update.
+      requestBody:
+        description: Data update.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '../../models/schemas/EstimateFee.yml'
+      responses:
+        200:
+          description: Estimated fee amount.
+          headers:
+            X-Id:
+              $ref: '../../models/headers/X-Id.yml'
+            X-Session-Token:
+              $ref: '../../models/headers/X-Session-Token.yml'
+            Request-Signature:
+              $ref: '../../models/headers/Request-Signature.yml'
+          content:
+            application/json:
+              schema:
```

Comment:
The response schema will be the fee estimation: EstimateFee

Reply:
@AlexBrandes Oh I see the mistake. I put the response schema in the wrong place. Updated.